### PR TITLE
Feat/2924 histoire support

### LINKED ISSUE #3293: Introduce Histoire

### PR DESCRIPTION
**Pull request by asvae**
closes #3293

Other:
* Fix broken tests

image

### CODE REVIEW

**Review by rustem-nasyrov — CHANGES_REQUESTED** [review]
Got infinite loading. None errors in the console.

**Comment by rustem-nasyrov** [comment]
@asvae
1. Checkout to the your branch.
2. yarn
3. yarn run story:dev (with phpstorm runner)
4. Open localhost:6006
5. Infinite loading

image
And two pending requests:
image

**Review by rustem-nasyrov — APPROVED** [review]

File: packages/ui/src/histoire/histoire.setup.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { defineSetupVue3 } from '@histoire/plugin-vue'
+
+import demoIconAliases from '../vue-book/vuestic-config/demo-icon-aliases'
+import demoIconFonts from '../vue-book/vuestic-config/demo-icon-fonts'
```
Comment (RVitaly1978) [inline packages/ui/src/histoire/histoire.setup.ts:4]: `vuestic-config` folder should be in `histoire` folder

**Review by RVitaly1978 — APPROVED** [review]

File: packages/ui/src/histoire/histoire.setup.ts (hunk 1 above)
Comment (asvae) [reply packages/ui/src/histoire/histoire.setup.ts:4]: That'll be handled as part of https://github.com/epicmaxco/vuestic-ui/issues/2924

> If it works we replace vue-book with Histoire.

Right now vue-book is still functional and I don't see a good reason to duplicate these configs.

**Comment by m0ksem** [comment]
@asvae, can we decline this PR?

**Comment by asvae** [comment]
Sure. Nothing to salvage here.

**Pull request closed without merging**